mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -0,0 +1,41 @@
+use crate::settings::Settings;
+use anyhow::Result;
+use tracing::info;
+
+pub struct NestedBridge {
```

Comment:
It would be great to add code comments from the beginning. Describe what is the purpose of the struct.